c52653092.lua

```diff
@@ -172,7 +172,10 @@ function c52653092.actop(e,tp,eg,ep,ev,re,r,rp)
 	e1:SetProperty(EFFECT_FLAG_PLAYER_TARGET)
 	e1:SetCode(EFFECT_CANNOT_ACTIVATE)
 	e1:SetTargetRange(0,1)
-	e1:SetValue(aux.TRUE)
+	e1:SetValue(c52653092.actlimit)
 	e1:SetReset(RESET_PHASE+PHASE_END)
 	Duel.RegisterEffect(e1,tp)
 end
+function c52653092.actlimit(e,re,tp)
+	return not re:GetHandler():IsImmuneToEffect(e)
```

Comment:
I think that Utopic ZEXAL is same as Outer Entity Azzathoth.
http://www.db.yugioh-card.com/yugiohdb/faq_search.action?ope=5&fid=10976&keyword=&tag=-1

> A.「合神竜ティマイオス」は『①：このカードは他のカードの効果を受けない』効果を持つモンスターです。
> したがって、「外神アザトート」の『相手はモンスターの効果を発動できない』効果が適用されているターンでも、「合神竜ティマイオス」はその「外神アザトート」の効果を受けず、『②：このカードが戦闘を行うダメージ計算時に発動できる。このカードの攻撃力・守備力は、フィールドのモンスターの一番高い攻撃力と同じになる』効果を通常通り発動する事ができます。